frontend/src/app/spot/styles/sass/common/container.sass

```diff
@@ -19,3 +19,6 @@
 
   > .spot-list
     margin: $spot-spacing-1 0
+
+  > .op-uc-container
+    padding-bottom: 0
```

Comment:
I think, we shouldn't put any "legacy" classes into our spot classes. this should go to `container-overrides.sass`